tests/containers/prepare_hpvs.pm

```diff
@@ -0,0 +1,171 @@
+# SUSE's openQA tests
+#
+# Copyright 2022-2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: IBM HPVS container runner
+#   A smoke test that BCI containers can run in IBM's Hyper Protect Platform
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use utils qw(script_retry file_content_replace);
+use version_utils;
+use testapi;
+use Mojo::JSON qw(decode_json);
+use serial_terminal 'select_serial_terminal';
+
+my $ibm = {
+    api => get_var('IBM_API'),
+    region => 'eu-gb',
```

Comment:
Will this change in the future?

Reply:
Possibly, the logging instance is in Frankfurt while the instance has to run (for now) in London as the s390x hpvs images are available there